scripts/anonymization/README.md

```diff
@@ -0,0 +1,220 @@
+
+| Name  | Default | Possible Value    | Description                                 |
+| ----- | ------- | ----------------- | ------------------------------------------- |
+| name  | -       | Any string        | The name of the column                      |
+| bins  | 10      | Any value         | The amount of distinct values in the output |
+| lower | -       | Any value         | Lower bound of the values                   |
+| upper | -       | Any value > lower | Upper bound of the values                   |
+
+---
+
+A full working example could look like this:
+
+```xml
+<anonymization>
+    <table name="item" epsilon="1.0" pre_epsilon="0.0" algorithm="aim">
+        <!-- Column categorization -->
+        <droppable>i_id</droppable>
+        <categorical>i_name,i_data,i_im_id</categorical>
+        <continuous>i_price</continuous>
+        <!-- Continuous column fine-tuning -->
+        <continuousConfig>
+            <column name="i_price" bins="1000" lower="2.0" upper="100.0">
+        <continuousConfig>
+    </table>
+</anonymization>
+```
+
+### Sensitive Values
+
+The anonymization module supports value faking to handle sensitive values. A `<sensitive>`-tag must be created to signify that column faking should be applied. Column faking can be defined individually on each column. To do so, parameters can be added inside of the `<column>`-tag. (See below)
+
+| Name    | Default | Possible Value                                   | Description                                                                                    |
+| ------- | ------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
+| name    | -       | Any string                                       | The name of the column                                                                         |
+| method  | pystr   | Any method provided by the python faker library  | The faking method of the library. If the method is not found, a random string will be inserted |
+| locales | -       | A string of supported locales separated by comma | The locale to produce localized values like chinese street names or english first names        |
+| seed    | 0       | Any integer value                                | The privacy budget spent on preprocessing of the data. Only necessary for continuous columns   |
+
+```xml
+<anonymization>
+    <table name="item" algorithm="aim">
+        <sensitive>
+            <column
+                name="i_name"
+                method="name"
+                locales="en_US"
+                seed="0">
+            </column>
+            <column name="i_data" method="pystr" />
+        </sensitive>
+    </table>
+</anonymization>
+```
+
+## Examples
+
+### Basic
+
+The most basic config will need only the name of the table and apply default values for the anonymization.
```

Comment:
Document current defaults, or where to look them up?

Reply:
Changed this part of the README. Automatic anonymization will automatically try to categorize columns. 

Defaults are documented and I provided a link to the algorithm repository as additional info